crds/tests/test_certify.py

```diff
@@ -843,6 +892,25 @@ def checksum_duplicate_rmap_case_error():
     >>> test_config.cleanup(old_state)
     """
 
+def checksum_roman_duplicate_rmap_case_error():
+    """
+    >>> from crds.refactoring import checksum
+    >>> old_state = test_config.setup(url="https://roman-crds-serverless.stsci.edu", observatory="roman")
+    >>> checksum.add_checksum("data/roman_wfi_flat_0004_duplicate.rmap")
+    CRDS - INFO -  Adding checksum for 'data/roman_wfi_flat_0004_duplicate.rmap'
+    CRDS - ERROR -  Duplicate entry at selector ('WFI01', 'F158') = UseAfter vs. UseAfter
+    >>> test_config.cleanup(old_state)
+    """
+
+def certify_roman_bad_tpn_value_rmap_case_error():
```

Comment:
I think this is meant to test an invalid keyword value in the rmap, yes? The name of the function reads to me like testing bad values in the tpn file. Something like certify_roman_invalid_rmap_tpn like certify_roman_invalid_asdf_tpn above.

Reply:
I will rename this today.

Reply:
Test renamed.